### PR TITLE
Implement size method

### PR DESCRIPTION
**Pull request by unnonouno**
I implemented `cupy.size` method. It does not generate a tuple but only returns `int`.
Note `numpy.size` exists but is not documented.

### CODE REVIEW

**Comment by unnonouno** [comment]
this function is not documented in numpy's document. Where is the best place to document in cupy?

File: cupy/core/core.pyx [hunk 1]
```diff
@@ -2636,6 +2636,29 @@ cdef _concatenate_kernel = ElementwiseKernel(
...
+        int: Number of elements along the given axis.
+
+    """
+    cdef int index
+    if axis is None:
+        return a.size
+    else:
+        index = axis
+        if index < 0:
+            index += a.ndim
```
Comment (okuta) [inline cupy/core/core.pyx]: How about using `self._shape.size()`?

**Comment by unnonouno** [comment]
I update it.

**Comment by okuta** [comment]
Thanks!

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**